libs/blocks/chart/chart.js

```diff
@@ -205,6 +206,67 @@ const areaSeriesOptions = (firstDataset) => (
   }))
 );
 
+export const setDonutLabel = (chart, label, unit = '', title = '') => {
+  chart?.setOption({ series: [{ label: { formatter: [`{a|${label?.toLocaleString()}${unit}}`, `{b|${title}}`].join('\n') } }] });
```

Comment:
curious - is there a chance that `setDonutLabel` is called with `chart === undefined || null` ?

Reply:
No not really. I removed it